lib/nvmf/rdma.c

```diff
@@ -363,6 +392,9 @@ struct spdk_nvmf_rdma_poll_group {
 /* Assuming rdma_cm uses just one protection domain per ibv_context. */
 struct spdk_nvmf_rdma_device {
 	struct ibv_device_attr			attr;
+#ifdef SPDK_CONFIG_NVMF_OFFLOAD
```

Comment:
I think, ibv_device_attr_ex shouldn't be depend on   SPDK_CONFIG_NVMF_OFFLOAD. We can reuse it for other features. 
It's better to add it under 
SPDK_EXTEND_OFED . Something like that.